Nodejs/Product/Nodejs/Project/NodejsProjectLauncher.cs

```diff
@@ -669,7 +669,6 @@ internal class NodePinezorroDebugLaunchConfig
         public object toPwaChromeServerConfig()
         {
             this.Console = "internalConsole";
-            this.SourceMapPathOverrides = new string[0];
```

Comment:
How come this is no longer necessary?

Reply:
Looks like it is! Adding it back for now, and taking a look at why it's required for us and not vscode.